src/main.cpp

```diff
@@ -11,8 +11,8 @@
 #include "chainparams.h"
 #include "checkpoints.h"
 #include "checkqueue.h"
-#include "consensus/consensus.h"
```

Comment:
No, you are right, it shouldn't be removed. I used to have the new declarations on consensus/consensus.h instead of consensus/tx_verify.h and I think I just replaced it in main like I did in other places (but in main it makes sense to have both).
Good catch, will change.